tests/Functional/BlobTest.php

```diff
@@ -67,6 +67,10 @@ public function testInsertNull(): void
 
     public function testInsertProcessesStream(): void
     {
+        if (! ini_get('allow_url_fopen')) {
```

Comment:
fixes phpunit test failures when `allow_url_open=0` is set in php.ini conf, example error:

```
fopen(): data:// wrapper is disabled in the server configuration by allow_url_fopen=0
```

Reply:
The requirement for `allow_url_fopen` is not specific to Windows and could be tested in any environment as follows:

```shell
php -dallow_url_fopen=Off vendor/bin/phpunit
```

This change will not be accepted due to the following. 

1. If we don't test with `allow_url_open=Off` on CI, there will be no way to no guarantee that after every change the test suite will remain passing in this configuration. So without that, this change is pointless.
2. We could rework the tests to not rely on `allow_url_fopen=On` but that would require moving all stream contents to separate files. This would make the test suite harder to maintain.
3. `allow_url_fopen` is enabled by default, and DBAL has no goal of having its test suite pass in any arbitrary configuration.

In order to run the test suite, you should enable `allow_url_fopen`.

Reply:
fixed by using `php://memory`